CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# CMakeLists.txt
+
+# Part of MR Lab Control Software
+
+# SPDX-License-Identifier: EUPL-1.2
+# SPDX-FileCopyrightText: Copyright (c) 2025 Martin Rumori/sonible GmbH
+
+cmake_minimum_required (VERSION 3.16)
+
+# This setting has to occur BEFORE the project call to work correctly
+set (CMAKE_OSX_DEPLOYMENT_TARGET "10.14" CACHE STRING "Minimum OS X deployment version" FORCE)
```

Comment:
I'd rephrase it to "macOS deployment target" – as that's the official OS name nowadays & a deployment target always indicates the minimum version